R/glm.R

```diff
@@ -0,0 +1,86 @@
+#' Axing a glm.
+#'
+#' glm objects are created from the base \pkg{stats} package.
+#'
+#' @inheritParams butcher
+#'
+#' @return Axed glm object.
+#'
+#' @examples
+#'
+#' cars_glm <- glm(mpg ~ ., data = mtcars)
+#' cleaned_glm <- butcher(cars_glm, verbose = TRUE)
+#'
+#' @name axe-glm
+NULL
+
+#' Remove the call.
+#'
+#' @rdname axe-glm
+#' @export
+axe_call.glm <- function(x, verbose = FALSE, ...) {
+  old <- x
+  x <- exchange(x, "call", call("dummy_call"))
+
+  add_butcher_attributes(
+    x,
+    old,
+    disabled = c("print()", "summary()"),
+    add_class = FALSE,
```

Comment:
I can't seem to figure out when we are supposed to add the class and when we aren't.

For example, ranger methods don't add the class:

https://github.com/tidymodels/butcher/blob/903847bd73c0edf8f55acba162c1f815cec121c0/R/ranger.R#L54-L65

But C5 methods do?

https://github.com/tidymodels/butcher/blob/903847bd73c0edf8f55acba162c1f815cec121c0/R/c5.R#L49-L59

I feel like we should add the class? Whatever we decide, let's be consistent with all glm methods in this file (it seems like the others add the class)

Reply:
I also can't understand when we do/do not add the class.